src/generators/json-all/index.mjs

```diff
@@ -0,0 +1,79 @@
+// @ts-check
+'use strict';
+
+import { writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+
+import { BASE_URL, DOC_NODE_VERSION } from '../../constants.mjs';
+import { generateJsonSchema } from './util/generateJsonSchema.mjs';
+
+/**
+ * This generator is responsible for collecting the JSON output generated by
+ * the `json` generator into a single JSON file.
+ *
+ * @typedef {Array<ApiDocMetadataEntry>} Input
+ *
+ * @type {GeneratorMetadata<Input, object>}
+ */
+export default {
+  name: 'json-all',
+
+  // This should be kept in sync with the JSON schema version for this
+  // generator AND the `json` generator
+  version: '2.0.0',
+
+  description:
+    'This generator is responsible for collecting the JSON output generated by the `json` generator into a single JSON file.',
+
+  dependsOn: 'json',
+
+  /**
+   * Generates a JSON file.
+   *
+   * @param {Input} input
+   * @param {Partial<GeneratorOptions>} param1
+   * @returns {Promise<object>}
+   */
+  async generate(input, { version, output }) {
+    const generatedValue = {
+      $schema: `${BASE_URL}/docs/${DOC_NODE_VERSION}/api/node-doc-all-schema.jsonc`,
```

Comment:
Can we make a SCHEMA_URL constant?

Reply:
Not really since it intakes the version given by the generator options, same for the rest of the schema url instances